npcs/guild/Penelope/penelope.md

```diff
@@ -1,3 +1,5 @@
 # Penelope
 
 Penelope is a young woman who looks late teens to early twenties, though she acts quite childish. She is "under the guild's protection", per Wendy. She was raised by the Syndicate. [Runt](runt) freed and hide her from them until guildmembers later retrieved and brought her to the guild.
+
```

Comment:
Why add these newlines?

Reply:
I was working on adding invisible text for Penelope's page. Something thematic about her illusion magic. In the end I took it out since I couldn't figure out the CSS.